apps/server/src/modules/dispositif/dispositif.service.ts

```diff
@@ -85,6 +88,7 @@ export enum NotifType {
   PUBLISHED = "PUBLISHED",
   DELETED = "DELETED",
   UPDATED = "UPDATED",
+  TO_VALIDATE = "TO_VALIDATE",
   UPDATED_AND_PUBLISHED = "UPDATED_AND_PUBLISHED",
```

Comment:
Techniquement maintenant ce pourrait être:

```suggestion
  VALIDATED_AND_PUBLISHED = "VALIDATED_AND_PUBLISHED",
```

Dans ce cas il faudrait également une migration pour passer mettre à jour tous les status `UPDATED_AND_PUBLISHED` à `VALIDATED_AND_PUBLISHED` dans mongo.

Reply:
En effet c'est plus correct, j'ai changé. En revanche pas besoin de migration, c'est seulement un statut utilisé par une fonction, mais il n'est pas présent en base